src/workstation-gui/config/win_server.bat

```diff
@@ -0,0 +1,6 @@
+@echo off
+start chrome http://stackoverflow.com/questions/tagged/python+or+sql+or+sqlite+or+plsql+or+oracle+or+windows-7+or+cmd+or+excel+or+access+or+vba+or+excel-vba+or+access-vba?sort=newest
+start "" c:\windows\write.exe
+start "" c:\windows\notepad.exe
```

Comment:
are these commands essential?

Reply:
No, just for testing I have remove these